examples/cpp/PineAPPL.hpp

```diff
@@ -0,0 +1,257 @@
+/**
+ * @file PineAPPL.hpp
+ * @brief object oriented interface to PineAPPL
+ */
+#ifndef PineAPPL_HPP_
+#define PineAPPL_HPP_
+
+#include <string>
+#include <list>
+#include <LHAPDF/LHAPDF.h>
+
+#include <pineappl_capi.h>
+
+/** @brief object oriented interface to PineAPPL  */
+namespace PineAPPL {
+
+/** @brief Key-value storage for passing optional information during grid creation */
+struct KeyVal {
```

Comment:
I suggest to use `struct` for PODs (like `KeyVal` and `Order`) and `class` otherwise.

Reply:
AFAIK, the only difference between `struct` and `class` is the default scope - since here everything is public I'd like to keep `struct` (thus saving one line)